packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -37,10 +39,21 @@ export const ArticlePostCard = forwardRef(function PostCard(
   }: PostCardProps,
   ref: Ref<HTMLElement>,
 ): ReactElement {
+  const { data } = useBlockPost(post);
   const onPostCardClick = () => onPostClick(post);
   const { trending, pinnedAt } = post;
   const customStyle = !showImage ? { minHeight: '15.125rem' } : {};
 
+  if (data?.showTagsPanel && post.tags.length > 0) {
+    return (
+      <PostTagsPanel
+        className="overflow-hidden h-full max-h-[23.5rem]"
+        post={post}
+        toastOnSuccess
```

Comment:
What makes this difference for a list or card?

Reply:
They should be the same, the only differences would probably the className.

Reply:
The one has `toastOnSuccess` the other doesn't please check if that's alright then 🙏

Reply:
Oh, might have been missed. Yep, they should both have the same as the don't have the panel to Undo the changes.